### PR TITLE
nRF52 OTA documentation

### PR DESCRIPTION
**Pull request by pdxlocations**
This PR adds the OTA firmware update methods for nRF52 devices.
[preview link](https://sandbox-git-ota-update-pdxlocations.vercel.app/docs/getting-started/flashing-firmware/nrf52/)

### CODE REVIEW

**Comment by garthvh** [comment]
Can apple be the selected tab since it actually works?

**Comment by pdxlocations** [comment]
> Can apple be the selected tab since it actually works?

@garthvh Yes that's smart.  (I use my wife's iPhone to update my nRF's!)

**Comment by charminULTRA** [comment]
Android tab: "nRF DFU app (v2.3.0)" - Can you specify this is the **Android** nRF DFU app?

What's the difference between the Android DFU App and the Android Connect app?

**Review by charminULTRA — CHANGES_REQUESTED** [review]

File: docs/getting-started/flashing-firmware/nrf52/ota.mdx
[the remark this answers is not in the document]
Comment (pdxlocations) [reply docs/getting-started/flashing-firmware/nrf52/ota.mdx:1]: The DFU App is MUCH simpler to use and the current release works on Apple.  The only app that works on Android (for now) is an old version of the Connect app which has to be manually installed from GitHub.  Maybe that could be a little clearer here.

**Comment by pdxlocations** [comment]
@charminULTRA I gave it a shot.

**Review by garthvh — APPROVED** [review]

File: docs/getting-started/flashing-firmware/nrf52/ota.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+{label: 'Apple', value: 'apple'},
+]}>
+
+<TabItem value="android">
+
+:::info
+As of this writing, the current Android release of the nRF DFU app (v2.3.0) is not compatible with Meshtastic firmware updates.
+:::
+
+OTA firmware updates are available for Android using and older release of the more advanced nRF Connect App __version 4.24.3__ which is available for download from the [Nordic Semiconductor GitHub page](https://github.com/NordicSemiconductor/Android-nRF-Connect/releases/tag/v4.24.3).
```
Comment (charminULTRA) [inline docs/getting-started/flashing-firmware/nrf52/ota.mdx:27]: Small typo, says "using and older"

**Review by charminULTRA — CHANGES_REQUESTED** [review]
There's a small typo, I left a comment for the location

**Comment by pdxlocations** [comment]
@charminULTRA good catch! fixed.

**Review by charminULTRA — APPROVED** [review]

**Pull request merged**